cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -1246,6 +1246,33 @@ const char* convert_fromUTF8_binary(gdv_int64 context, const char* bin_in, gdv_i
   return ret;
 }
 
+FORCE_INLINE
+const char* convert_replace_invalid_fromUTF8_binary(
+    gdv_int64 context, const char* text_in, gdv_int32 text_len,
+    const char* char_to_replace, gdv_int32 /*char_to_replace_len*/, gdv_int32* out_len) {
+  *out_len = text_len;
+  char* ret = reinterpret_cast<char*>(gdv_fn_context_arena_malloc(context, *out_len));
```

Comment:
output string length can be larger than input binary length

Reply:
As defined by email, actually we are going to consider a single char to replace. So the input length will necessarily be equals to the output length!